index.html

```diff
@@ -23,8 +23,8 @@
                 <span class="line2"></span>
                 <span class="line3"></span>
             </div>
-            <a href="#news">Portfolio</a>            
```

Comment:
- [ ] As I can see you did great work, kindly make sure to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

    - As you can see the Figma design, your margins are note exactly the some 
    - Kindly, make sure your `header` look like the Figma design.  
   
> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |<img width="226" alt="mobile 2 0" src="https://user-images.githubusercontent.com/57408419/219794036-a17e4400-fc73-46e6-af4e-b3f3d99aa41b.png"> | | ![01](https://user-images.githubusercontent.com/57408419/219794667-0b13becd-741c-4484-aee5-77623e478fd9.jpg) | |

- [ ] Kindly, make sure you remove the green background 👍 
- [ ] Kindly, make sure your images are running 👍 
- [ ]  Kindly, make sure you fixed margins and look like the Figma design

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![03](https://user-images.githubusercontent.com/57408419/219798145-e1210d2b-fa27-4d56-bc1f-3141947c6044.jpg) | | ![02](https://user-images.githubusercontent.com/57408419/219798234-a123828a-fc8c-4662-bb83-4b8832aa1c21.jpg) | |



- [ ] Kindly, make sure you fixed the language images and fix the flex-direction for languages and Frameworks 👍 
- [ ] Kindly, make sure you fixed the margins 👍 


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![05](https://user-images.githubusercontent.com/57408419/219800575-c4f47169-8ea7-418d-90da-2abe1234d611.jpg) | | ![04](https://user-images.githubusercontent.com/57408419/219800648-bececbea-867a-46c8-ac87-815cf3d247b8.jpg) | |